### ISSUE TITLE
is there a limit amount of log messages that I can display in Report Portal?

### ISSUE THREAD

**Issue by OrenAutoSmileBox** [reportportal/reportportal#760]
when I reach page 9 on lunches section
(i display in one page the maximum amount of logs that I can 300)
Report portal stops displaying log messages

is there a limit on the number of log messages that Report Portal displays?
can I configure it?

**Comment by DzmitryHumianiuk** [comment]
@OrenAutoSmileBox there is no any limitation both for logs and for launches.

just check if your VM with ReportPortal has enough resources (CPU and RAM) to handle app + data in db.

**Comment by OrenAutoSmileBox** [comment]
thanks for the input

but I see no problem with the resources
the machine that runs the RP containers have
4 cores
16g RAM
40g storage
and currently, 20g storage occupied

I didn't set any limitation on the container's resource consumption


if you have any additional input 
I would like to hear it
because I don't know what causing this limitation

**Comment by DzmitryHumianiuk** [comment]
@OrenAutoSmileBox what is the version of application? 
(copy under login form)

**Comment by OrenAutoSmileBox** [comment]
Current version: API Service: 4.3.4;
Index Service: 4.2.0;
Jira Service: 4.3.0;
Rally Service: 4.3.0;
Authorization Service: 4.2.0;
Service UI: 4.3.6;

**Comment by DzmitryHumianiuk** [comment]
@OrenAutoSmileBox 
What's the MongoDB version behind it?

btw. we are about to release v5.
v4 is not longer supported.

**Comment by OrenAutoSmileBox** [comment]
![containers](https://user-images.githubusercontent.com/52413954/70056597-66d23380-15e4-11ea-9b9d-24b9f292125c.png)


MongoDB: 3.4

**Comment by DzmitryHumianiuk** [comment]
@OrenAutoSmileBox can you take a look into the logs at the moment when you reach page 9?
Logs from `service-api` container.

Since we use the same images to run our production instance with 1.5TB of data in it, 150+ projects and 700+ active users a day. And don't face any kind of that issues.

our versions
```
Analysis Service: 4.3.0; 
API Service: 4.3.11; 
Index Service: 4.2.0; 
Jira Service: 4.3.0; 
Rally Service: 4.3.0; 
Authorization Service: 4.2.0; 
Service UI: 4.3.7;
```

**Comment by DzmitryHumianiuk** [comment]
closed with respect to v5.0

please verify in new version, and open new issue if still actual
